ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -151,8 +151,12 @@ const TopologyComponent = ({ model }: TopologyComponentProps) => {
                             edges={model?.edges || []}
                         />
                     )}
-                    {selectedEntity && selectedEntity?.data?.type === 'CIDR_BLOCK' && (
-                        <CidrBlockSideBar />
+                    {selectedEntity && selectedEntity?.data?.type === 'EXTERNAL_SOURCE' && (
```

Comment:
Yea you have a good point. I made the change because internally we kept it as the backend values and only used the UI values in the rendered view so I made the change to allow myself to see things for now. I will change it to `CIDR_BLOCK` in a future PR. We will eventually type out the `data` field so there is a more consistent data structure